src/USER-PLUMED/fix_plumed.cpp

```diff
@@ -0,0 +1,466 @@
+/* ----------------------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   http://lammps.sandia.gov, Sandia National Laboratories
+   Steve Plimpton, sjplimp@sandia.gov
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+
+/* ----------------------------------------------------------------------
+   Contributing author: Gareth Tribello (Queens U, Belfast)
+------------------------------------------------------------------------- */
+
+#include <cmath>
+#include <cstdlib>
+#include <cstring>
+
+#include "atom.h"
+#include "comm.h"
+#include "update.h"
+#include "force.h"
+#include "respa.h"
+#include "domain.h"
+#include "error.h"
+#include "group.h"
+#include "fix_plumed.h"
+#include "universe.h"
+#include "compute.h"
+#include "modify.h"
+#include "pair.h"
+
+/*
+  Use statically linked C++ interface to plumed
+*/
+#define __PLUMED_WRAPPER_CXX 1
+
+#include "plumed/wrapper/Plumed.h"
+
+/* -------------------------------------------------------------------- */
+
+using namespace LAMMPS_NS;
+using namespace FixConst;
+
+#define INVOKED_SCALAR 1
+
+FixPlumed::FixPlumed(LAMMPS *lmp, int narg, char **arg) :
+  Fix(lmp, narg, arg),
+  p(NULL), nlocal(0), gatindex(NULL), masses(NULL), charges(NULL),
+  id_pe(NULL), id_press(NULL)
+{
+
+  if (!atom->tag_enable)
+    error->all(FLERR,"Fix plumed requires atom tags");
+
+  if (igroup != 0 && comm->me == 0)
```

Comment:
Here is the new code to add a warning if the fix does not use group 'all' and print a reminded, that the group is ignored.